sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -0,0 +1,897 @@
+
+            return new DataFeed(name, dataSource, granularity, schema, ingestionSettings);
+        }
+
+        private DataFeed GetDataFeedWithOptionalMembersSet(string name, DataFeedSource dataSource)
+        {
+            var metrics = new List<DataFeedMetric>()
+            {
+                new ("cost") { MetricDisplayName = "costDisplayName", MetricDescription = "costDescription" },
+                new ("revenue") { MetricDisplayName = "revenueDisplayName", MetricDescription = "revenueDescription" }
+            };
+            var dimensionColumns = new List<DataFeedDimension>()
+            {
+                new ("city"),
+                new ("category") { DimensionDisplayName = "categoryDisplayName" }
+            };
+            var ingestionStartTime = DateTimeOffset.Parse("2020-08-01T00:00:00Z");
+
+            var granularity = new DataFeedGranularity(DataFeedGranularityType.Custom) { CustomGranularityValue = 1360 };
+            var schema = new DataFeedSchema(metrics) { DimensionColumns = dimensionColumns, TimestampColumn = "timestamp" };
+            var ingestionSettings = new DataFeedIngestionSettings(ingestionStartTime)
+            {
+                IngestionStartOffset = TimeSpan.FromMinutes(30),
+                IngestionRetryDelay = TimeSpan.FromSeconds(80),
+                StopRetryAfter = TimeSpan.FromMinutes(10),
+                DataSourceRequestConcurrency = 5
+            };
+
+            return new DataFeed(name, dataSource, granularity, schema, ingestionSettings)
+            {
+                Description = "This data feed was created to test the .NET client.",
+                AccessMode = DataFeedAccessMode.Public,
+                ActionLinkTemplate = "https://fakeurl.com/%metric/%datafeed",
+                Administrators = new List<string>() { "fake@admin.com" },
+                Viewers = new List<string>() { "fake@viewer.com" },
+                MissingDataPointFillSettings = new () { FillType = DataFeedMissingDataPointFillType.CustomValue, CustomFillValue = 45.0 }
+            };
+        }
+
+        private void ValidateDataFeedWithMinimumSetup(DataFeed dataFeed, string expectedId, string expectedName)
+        {
+            var ingestionStartTime = DateTimeOffset.Parse("2020-08-01T00:00:00Z");
+
+            Assert.That(dataFeed.Id, Is.EqualTo(expectedId));
+            Assert.That(dataFeed.Name, Is.EqualTo(expectedName));
+            Assert.That(dataFeed.Description, Is.Not.Null.And.Empty);
+            Assert.That(dataFeed.Status, Is.EqualTo(DataFeedStatus.Active));
+            Assert.That(dataFeed.AccessMode, Is.EqualTo(DataFeedAccessMode.Private));
+            Assert.That(dataFeed.ActionLinkTemplate, Is.Not.Null.And.Empty);
+            Assert.That(dataFeed.Creator, Is.Not.Null.And.Not.Empty);
+            Assert.That(dataFeed.Administrators, Is.Not.Null);
+            Assert.That(dataFeed.Administrators.Single(), Is.EqualTo(dataFeed.Creator));
```

Comment:
I don't see administrators being created for the minimum setup. Where does the information comes from?

Reply:
The creator is always set as admin.

Reply:
And where is the creator set? sorry if I missed it

Reply:
No prob. Do you remember we have [two authentication keys](https://github.com/Azure/azure-sdk-for-net/blob/bb740fbf6c471e206ca4710a273e3e3b640d5244/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorKeyCredential.cs#L27)? The Subscription Key is used the same way as the key in other Cognitive Services (you get it from the Azure Portal and it gives you the "right" to use the resource). The API Key is used to identify different users who use the same resource.

Given the API Key, the service is already able to identify who the creator of the Data Feed is, so the `Creator` property is attributed automatically. It can't be manually set nor changed.